pom.xml

```diff
@@ -108,6 +108,11 @@
             <artifactId>maven-model</artifactId>
             <version>3.3.9</version>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```

Comment:
I can shade it using this maven plugin https://maven.apache.org/components/plugins/maven-shade-plugin/